src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+    private final ColorSupplier colorSupplier;
```

Comment:
To avoid adding unnecessary ColorSupplier and Random in Application class, let's initialise them here 
```suggestion
    private final ColorSupplier colorSupplier = new ColorSupplier();
    private final Random randomGenerator = new Random();
```